src/global/Constants.h

```diff
@@ -12,9 +12,13 @@
 #include <string>
 
 #include "../util/Parameters.h"
+#include "util/MemorySize/MemorySize.h"
 
-static const size_t DEFAULT_STXXL_MEMORY_IN_BYTES = 5'000'000'000UL;
-static const size_t STXXL_DISK_SIZE_INDEX_BUILDER = 1000;  // In MB.
+// For access to `memorySize` literals.
+using namespace ad_utility::memory_literals;
+
+static const ad_utility::MemorySize DEFAULT_STXXL_MEMORY = 5_GB;
+static const ad_utility::MemorySize STXXL_DISK_SIZE_INDEX_BUILDER = 1000_MB;
 static const size_t STXXL_DISK_SIZE_INDEX_TEST = 10;
```

Comment:
This should also be `MemorySize`.

Reply:
After a short search, I found out, that **nobody** actually uses this variable. So, I deleted it in [5fb137a](https://github.com/ad-freiburg/qlever/pull/1106/commits/5fb137af2450d40ce4978bf2c3cbd97f278f6197).